### DISCUSSION TITLE
스터디, 프로젝트 관련 테이블 수정

### DISCUSSION THREAD

**Discussion by siwan9** [Kumoh-talk/kumoh-talk-Backend#58] [Show and tell]
![image](https://github.com/user-attachments/assets/b5dc0a4b-35e5-4ae9-b061-327ef2dcbc0c)

기존 Json을 그대로 넣는 것보다 해당 구조로 테이블을 구성하면 어떨까 해서 수정해보았습니다.
문제점이나 다른 좋은 수정 방향 의견 주시면 감사하겠습니다!
별 다른 문제점 없으면 해당 테이블을 토대로 기능 개발 진행하겠습니다.

**Comment by kang20** [comment]
신청서 객관식 선택지는 모집글을 올린 사용자가 올린 답 보기인거죠?

**Reply by siwan9** [reply]
넵 맞습니다
